src/commands/perms.js

```diff
@@ -9,10 +9,10 @@ const permissions = [`CREATE_INSTANT_INVITE`, `KICK_MEMBERS`, `BAN_MEMBERS`, `MO
 `VIEW_GUILD_INSIGHTS`, `READ_MESSAGE_HISTORY`, `SEND_MESSAGES`, `SEND_TTS_MESSAGES`, `ATTACH_FILES`, `USE_EXTERNAL_STICKERS`, `ADD_REACTIONS`, `SEND_MESSAGES_IN_THREADS`,
 `CREATE_PUBLIC_THREADS`, `CONNECT`, `SPEAK`, `PRIORITY_SPEAKER`, `STREAM`, `MUTE_MEMBERS`, `DEAFEN_MEMBERS`, `MOVE_MEMBERS`, `MENTION_EVERYONE`];
 
-let permsOutput = `\`\`\`\nServer - 📛\nCurrent channel - ♨️\n\n📛 | ♨️\n`;
+let permsOutput = `\`\`\`\nUser permissions in:\n  Server: 📛\n  Current channel: ♨️\n\n📛 | ♨️\n`;
```

Comment:
Remove spaces after \n's

Reply:
makes it a lot clearer but ok done

Reply:
@Eliiia No it messes up how it looks on mobile im pretty sure